tests/LogicItLab/Salesforce/MapperBundle/WsdlValidatorTestCase.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace LogicItLab\Salesforce\MapperBundle;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class WsdlValidatorTestCase extends KernelTestCase
+{
+    /** @var WsdlValidator */
+    private $wsdlValidator =  null;
+
+    public function testWsdlIsValid()
+    {
+        $this->assertInstanceOf(WsdlValidator::class, $this->wsdlValidator, "Validator should be booted before run a test.");
+        $this->assertEmpty($this->wsdlValidator->retrieveMissingFields());
```

Comment:
What do you think of renaming `retrieveMissingFields` to `validate`? As it is a validator it would represent its main feature.